Makefile

```diff
@@ -196,13 +199,24 @@ install: $(DEBUG_APK) ## Install the debug APK on a connected device
 run: install ## Run the debug APK on a connected device
 	adb shell am start -n com.tailscale.ipn/com.tailscale.ipn.MainActivity
 
-.PHONY: dockershell
-dockershell: ## Run a shell in the Docker build container
```

Comment:
What is the replacement for this command? does it have no users anymore?

Reply:
Added back.  There's a legit use for it.